Soccar [Game-Server]/src/nl/soccar/gameserver/model/GameServer.java

```diff
@@ -26,7 +28,7 @@ private GameServer() {
     }
 
     public static GameServer getInstance() {
-        return instance;
+        return INSTANCE;
     }
 
     public void registerPlayer(PlayerWrapper player) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)